### ISSUE TITLE
Issue when creating a new project in an existing directory

### ISSUE THREAD

**Issue by shauunhimseelf** [shedcollective/shed-cli-tool#1]
**What happened**
When creating a project I tried to use the option to create in current directory

Got this error
```
Project Name: ktsp website
Project Slug: [ktsp-website]
Project Directory (Leave blank for current directory):
Backend Framework? [Static]
  [0] Static
  [1] Laravel
  [2] Nails
  [3] WordPress
 > 2
Frontend Framework? [None]
  [0] None
  [1] Webpack
 > 1

Does this all look OK?

Project Name:  ktsp website
Project Slug:  ktsp-website
Directory:  /Users/shaun/Sites/GitRepos/ktsp-website/
Backend Framework:  Nails
Frontend Framework:  Webpack

Continue?

📁 Creating directory /Users/shaun/Sites/GitRepos/ktsp-website/...
Warning: mkdir(): File exists in phar:///usr/local/Cellar/shed/1.0.8/bin/shed/src/Command/Project/Create.php on line 456

In Create.php line 457:

  [Shed\Cli\Exceptions\Directory\FailedToCreateException]


project:create [-p|--name [NAME]] [-s|--slug [SLUG]] [-d|--directory [DIRECTORY]] [-b|--backend-framework [BACKEND-FRAMEWORK]] [-f|--frontend-framework [FRONTEND-FRAMEWORK]]
```

**Comment by hellopablo** [comment]
Can recreate. #willfix

**Comment by hellopablo** [comment]
Note to self; the method creating the folder is not checking properly for the existence of the folder first.

**Comment by hellopablo** [comment]
Fixed in 1.1.2
